src/main/java/seedu/address/logic/commands/EditSupplierCommand.java

```diff
@@ -96,9 +96,10 @@ public CommandResult execute(Model model) throws CommandException {
     /**
      * combine the offer set of supplierToEdit with EditSupplierDescriptor
      * return the combined set
+     * @return
```

Comment:
Can you fix this javadoc?

Maybe 
~~~java
/**
 * Combines the two sets of offers given.
 * @return the combined set
 */
~~~

Reply:
Then, you can change the input names to a and b, since it does not matter where the sets come from in the function.